src/indicators/flooding/glofas/area.py

```diff
@@ -0,0 +1,47 @@
+from collections import namedtuple
+from typing import Dict
+
+from shapely.geometry import Polygon
+
+Station = namedtuple("Station", "lon lat")
+
+
+class Area:
+    def __init__(self, north, south, east, west):
+        self.north = north
+        self.south = south
+        self.east = east
+        self.west = west
+
+    def list_for_api(self):
+        return [self.north, self.west, self.south, self.east]
+
+
+class AreaFromStations(Area):
+    def __init__(self, stations: Dict[str, Station], buffer: float = 0.5):
```

Comment:
I'm wondering if a default buffer of 0.5 degrees might be a bit larger than one would ever need?

Reply:
That's a good point, I'll change it to 0.1

Reply:
Oh yeah, fair! I didn't think about it in terms of GloFAS resolution.